app/models/workflow.rb

```diff
@@ -58,11 +58,11 @@ def self.set_up_the_method(step)
 
       events
         .reject { |x| x.message }
-        .each   { |e| e.message = mash_single_value(event_handler.config[:message], event) }
+        .each   { |e| e.message = mash_single_value(event_handler.config[:message], e) }
```

Comment:
I dont understand line 60 -- why would we filter out events that already have a message. Or rather, why would they have a message?

I just think we can get rid of 59-61 entirely because the message is gonna get set with `mash` anyway.

Not a big deal -- just trying to cross reference my understanding. There's a lot of new code to catch up on!

Reply:
> ...gonna get set with `mash` anyway...

obv assuming we remove the 'dont touch the message' filter in `mash`

Reply:
If I reject those events that have a message, it allows the handler to set a more specific message -- if it is programmed to.  But if is ignored, then this way of handling message becomes a default behavior.